app.js

```diff
@@ -23,8 +26,9 @@ app.use((err,req,res,next)=>{
 app.use((err,req,res,next)=>{
     if(err.status){
         res.status(err.status).send({msg: err.msg});
+    }else{
+        next(err)   
     }
-    next(err)
 })
```

Comment:
Maybe at this point you should consider putting your error handlers into a separate controller file?